wgengine/tsdns/forwarder.go

```diff
@@ -143,8 +143,9 @@ func (f *forwarder) Start() error {
 	}
 
 	f.wg.Add(connCount + 1)
-	for idx, conn := range f.conns {
-		go f.recv(uint16(idx), conn)
+	for idx := range f.conns {
+		idx := idx
```

Comment:
this whole change is unnecessary.

arguments to goroutines are evaluated before the goroutine starts.

Reply:
Sigh. Yes, thanks. I just misread.